galore/plot.py

```diff
@@ -145,20 +151,22 @@ def plot_pdos(pdos_data, ax=None, total=True, show_orbitals=True,
         else:
             x_data = el_data['energy'] + offset
 
-        orbitals = list(el_data.keys())
-        orbitals.remove('energy')
-
-        for orbital in orbitals:
-            if total:
-                tdos += el_data[orbital]
-            else:
+        for orbital in orbitals: # orbitals defined above
+            if not total:
                 max_y = max(max_y, max(el_data[orbital]))
 
             if show_orbitals:
-                ax.plot(x_data, el_data[orbital],
-                        label="{0}: {1}".format(element, orbital),
+                color = next(ax._get_lines.prop_cycler)['color'] # get color to ensure
+                # matching fill
+                label = None if max(
+                    el_data[orbital]) < (legend_cutoff/100)*tmax else \
```

Comment:
Not finding this inline if/else with an explicit linebreak to be very readable. The split on `max(` is also a bit misleading as it implies that the whole following block is inside the `max`.

If we define `legend_cutoff` to be a fraction (which is nicer anyway!) then
```python
                label = (None if max(el_data[orbital]) < legend_cutoff * tmax
                         else "{0}: {1}".format(element, orbital))
```
will fit. If for some reason we go with percentages, I still think it's prettier to split on operators with parentheses.

```python
                label = (None if (max(el_data[orbital])
                                  < (tmax * legend_cutoff / 100))
                         else "{0}: {1}".format(element, orbital))
```

Reply:
Done in 0f58b3e4ee52fdd27807db18d0753160aa7525a6